<launch>

<node pkg="rosbag" type="play" name="player" output="screen" args="$(dirname)/project.bag"/>

 <group ns="front">
    <node pkg="first_project" name="front" type="enu"/>
        <param name="latitude_init"  value="45.6216561271" />
        <param name="longitude_init"  value="9.28155230131" />
        <param name="h0" value="224.616616895" />

 </group>

<group ns="obs">
    <node pkg="first_project" name="obs" type="enu" />
        <param name="latitude_init" value="45.6216561271" />
        <param name="longitude_init" value="9.28155230131" />
        <param name="h0" value="224.616616895" />


 </group>
    
     <node pkg="first_project" name="collector" type="collector"/>
        <param name="safe" value="5" />
        <param name="unsafe" value="1" />

  <node pkg="first_project" name="distance_calc" type="distance_calc"/>

  <node pkg="rqt_reconfigure" name="rqt_reconfigure" type="rqt_reconfigure"/>
  
</launch>
